server/expression/reflection.go

```diff
@@ -0,0 +1,90 @@
+package expression
+
+import (
+	"fmt"
+)
+
+type ReflectionToken struct {
```

Comment:
I'm having issues connecting the `reflection` name to what is happening here, what would you say is the main goal of this module?

Reply:
Makes sense. My idea was: I'm inspecting the result of an expression and usually this is called reflection. But just calling them tokens might make sense.

Reply:
Renamed it to `Token` instead.